quay-enterprise/quay-ssl.md

```diff
@@ -0,0 +1,136 @@
+# SSL + Quay Enterprise
+
+**NOTE**: This document assumes you have deployed [Quay Enterprise as a single container.](https://tectonic.com/quay-enterprise/docs/latest/initial-setup.html)
+
+Quay Enterprise will be configured with a [self-signed certificate.](https://en.wikipedia.org/wiki/Self-signed_certificate) A Certificate Authority is required.
+
+First, create a rootCA:
+
+```
+$ openssl genrsa -out rootCA.key 2048
+$ openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 1024 -out rootCA.pem
+```
+
+Next create the key and certificate that will be signed by the CA:
+
+```
+$ openssl genrsa -out ssl.key 2048
+```
+
+When creating the `ssl.csr` file it is important that the hostname of the server where QE is installed is used as the `Common Name` or QE will reject the  configuration. In this demo environment QE is currently installed at `lan-lab-07.lab.libcore.so`
+
+
+```
+$ openssl req -new -key ssl.key -out ssl.csr
+
+-----
+Country Name (2 letter code) [XX]:US
+State or Province Name (full name) []:California
+Locality Name (eg, city) [Default City]:SF
+Organization Name (eg, company) [Default Company Ltd]:Demo Quay
+Organizational Unit Name (eg, section) []:Demo Quay
+Common Name (eg, your name or your server's hostname) []:lan-lab-07.lab.libcore.so
+Email Address []:support@quay.io
+```
+
+Sign the certificate with the CA:
+
+```
+$ openssl x509 -req -in ssl.csr -CA rootCA.pem -CAkey rootCA.key -CAcreateserial -out ssl.cert -days 500 -sha256
+Signature ok
+subject=/C=US/ST=California/L=SF/O=Demo Quay/OU=Demo Quay/CN=lan-lab-07.lab.libcore.so/emailAddress=support@quay.io
+Getting CA Private Key
+```
+The next step can be accomplished via the QE superuser panel or from the terminal:
+
+# For setting up via superuser panel:
```

Comment:
Heading hierarchy, see: https://github.com/coreos/docs/blob/master/STYLE.md#heading-semantics-and-the-sidebar-outline

This is at least an <h2> heading:
`## To configure with the superuser GUI in QE:`